src/BusinessNotificationCenter.jsx

```diff
@@ -0,0 +1,120 @@
+import './App.css';
+import { useEffect, useState } from 'react';
+import { useBackend } from './contexts/BackendContext';
+import { SearchIcon, PlusSquareIcon, BellIcon, HamburgerIcon } from '@chakra-ui/icons';
+import {
+  Text,
+  InputGroup,
+  InputLeftElement,
+  Icon,
+  Menu,
+  MenuButton,
+  Button,
+  Input,
+  Table,
+  Thead,
+  Tbody,
+  Tr,
+  Th,
+  Td,
+} from '@chakra-ui/react';
+import {
+  Modal,
+  ModalOverlay,
+  ModalContent,
+  ModalHeader,
+  ModalFooter,
+  ModalBody,
+  ModalCloseButton,
+  useDisclosure,
+} from '@chakra-ui/react';
+
+const App = () => {
+  const { isOpen, onOpen, onClose } = useDisclosure();
+  const { backend } = useBackend();
+  const [data, setData] = useState([]);
+
+  const getData = async () => {
+    try {
+      const id = '5';
```

Comment:
good job using a const but maybe identify this id at the top of your file and indicate that it should not be hard coded in the future for when we setup the authentication (the authentication will enable us to tell the file what the value of the business_id is.

also call it business_id not just id for clarity